### ISSUE TITLE
how to handle metadata

### ISSUE THREAD

**Issue by lzehl** [INCF/neuroscience-data-structure#3]
This issue should discuss how metadata should be handled in neuroscience-data-structure (NDS).
Within this issue we should discuss :
- metadata schemas, e.g.:
  - https://github.com/G-Node/odml-terminologies
  - http://neuroshapes.org/
- what is required, what is optional, what can be additionally provided?

How this is handled in BIDS:
- preferred formats: TSV and JSON
- schemas: are defined in the specs and somewhere in the GitHub repo (I think)
Metadata properties can be required or optional. Not sure if you can provide additional properties.

The matter of preferred metadata file formats is discussed in #2

**Comment by SDV96** [comment]
IS THIS ISSUE SOLVED I CAN IMPROVE IT By..... if @lzehl zehi approves  assign within 1 WEEKS {WORKING NOW}

**Comment by lzehl** [comment]
@SDV96 this issue became irrelevant when the NDS moved to become a BIDS extension proposal no?

**Comment by SylvainTakerkart** [comment]
> [@SDV96](https://github.com/SDV96) this issue became irrelevant when the NDS moved to become a BIDS extension proposal no?

yes! the issues on this repo were mostly used as a discussion forum around specific topics, that we then addressed in the BEP... I will close them to avoid confusion, and modify the readme file at the root of the repo...
